specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/2022-05-31/iotcentral.json

```diff
@@ -0,0 +1,3968 @@
+        },
+        "message": {
+          "type": "string",
+          "description": "Error message details."
+        },
+        "requestId": {
+          "type": "string",
+          "description": "Correlation Id for current request."
+        },
+        "time": {
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "description": "The time that error request failed."
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ]
+    },
+    "FileUpload": {
+      "description": "The file upload configuration definition.",
+      "type": "object",
+      "properties": {
+        "account": {
+          "description": "The storage account name where to upload the file to",
+          "type": "string"
+        },
+        "connectionString": {
+          "description": "The connection string used to configure the storage account",
+          "type": "string"
+        },
+        "container": {
+          "description": "The name of the container inside the storage account",
+          "type": "string"
+        },
+        "sasTtl": {
+          "description": "ISO 8601 duration standard, The amount of time the device’s request to upload a file is valid before it expires.",
+          "type": "string"
+        },
+        "state": {
+          "description": "The state of the file upload configuration",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "pending",
+            "updating",
+            "deleting",
+            "succeeded",
+            "failed"
+          ],
+          "x-ms-enum": {
+            "name": "FileUploadState",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "pending",
+                "name": "Pending"
+              },
+              {
+                "value": "updating",
+                "name": "Updating"
+              },
+              {
+                "value": "deleting",
+                "name": "Deleting"
+              },
+              {
+                "value": "succeeded",
+                "name": "Succeeded"
+              },
+              {
+                "value": "failed",
+                "name": "Failed"
+              }
+            ]
+          }
+        },
+        "etag": {
+          "description": "ETag used to prevent conflict with multiple uploads",
+          "type": "string"
+        }
+      },
+      "required": [
+        "connectionString",
+        "container"
+      ]
+    },
+    "Organization": {
+      "description": "The organization definition.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique ID of the organization.",
+          "type": "string",
+          "readOnly": true
+        },
+        "displayName": {
+          "description": "Display name of the organization.",
+          "type": "string"
+        },
+        "parent": {
```

Comment:
Just a suggestion. Would it be more precise to name it e.g. `parentId` or `parentOrganizationId` or `parentOrganizationName`?